integrationtests/volume_test.go

```diff
@@ -9,12 +9,10 @@ import (
 	"testing"
 	"time"
 
-	diskv1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1"
-	v1 "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1"
-	"github.com/kubernetes-csi/csi-proxy/client/api/volume/v1alpha1"
-	diskv1client "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1"
-	v1client "github.com/kubernetes-csi/csi-proxy/client/groups/volume/v1"
-	v1alpha1client "github.com/kubernetes-csi/csi-proxy/client/groups/volume/v1alpha1"
+	diskv1beta3 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta3"
```

Comment:
@jingxu97 @ddebroy please review this file, after reading some part of it it didn't make sense to use alpha apis anymore so I'm only using v1beta apis but I wanted to confirm if this is ok because there were some places where the alpha and the beta client were created explicitly (i.e. it was not a single client per api)